src/sonic-eventd/rsyslog_plugin/rsyslog_plugin.cpp

```diff
@@ -25,50 +25,77 @@ bool RsyslogPlugin::onMessage(string msg, lua_State* luaState) {
     }
 }
 
+void parseParams(vector<string> params, vector<EventParam>& eventParams) {
+    for(long unsigned int i = 0; i < params.size(); i++) {
+        if(params[i].empty()) { continue; }
```

Comment:
Is this even possible?

Reply:
This is just taking care of the edge case where in the regex file

params: [ '', '', '', ''] 

No need to do any type of parsing, just included for edge case

Reply:
Let us report error at least

Reply:
Done